### PR TITLE
Curating Owners ===============

### PR DESCRIPTION
**Pull request by apelisse**
In an effort to expand the existing pool of reviewers and establish a
two-tiered review process (first someone lgtms and then someone
experienced in the project approves), we are adding new reviewers to
existing owners files.
## If You Care About the Process:

We did this by algorithmically figuring out who’s contributed code to
the project and in what directories.  Unfortunately, that doesn’t work
well: people that have made mechanical code changes (e.g change the
copyright header across all directories) end up as reviewers in lots of
places.

Instead of using pure commit data, we generated an excessively large
list of reviewers and pruned based on all time commit data, recent
commit data and review data (number of PRs commented on).

At this point we have a decent list of reviewers, but it needs one last
pass for fine tuning.
## TLDR:

As an owner of a sig/directory and a leader of the project, here’s what
we need from you:
1. Comment on the PR with the list of names you want removed from the
   reviewers list (please, make one name per line, just the name so it's
   easy to process automatically).

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/35711)

### CODE REVIEW

**Comment by grodrigues3** [comment]
Test PR
